vscode/package.json

```diff
@@ -140,6 +140,11 @@
         "title": "Ruby file operations",
         "category": "Ruby LSP",
         "icon": "$(ruby)"
+      },
+      {
+        "command": "rubyLsp.diagnoseIssues",
+        "title": "Diagnose issues",
```

Comment:
Currently, the command only outputs information about the workspace and editor configuration. Do you foresee we using this for any automatic diagnostic of problems? And if not, would a different term be clearer?

Reply:
How about something along the lines of:
- Display diagnostics
- Display de debugging informations (this one might be confusing with the actual debugger)
- Show environment informations

Or would that be too specific?